README.md

```diff
@@ -1,6 +1,22 @@
 # mathlib-tools
 
-Supporting tools for [Lean mathlib](https://leanprover-community.github.io/).
-
 [![Build Status](https://travis-ci.org/leanprover-community/mathlib-tools.svg?branch=master)](https://travis-ci.org/leanprover-community/mathlib-tools)
 [![Build status](https://ci.appveyor.com/api/projects/status/t353pkb62tep1rth?svg=true)](https://ci.appveyor.com/project/cipher1024/mathlib-tools)
+
+This package contains several supporting tools for [Lean mathlib](https://leanprover-community.github.io/).
+
+  - `update-mathlib` fetches precompiled olean files if you use mathlib as a dependency
+  - `cache-olean` fetches precompiled olean files if you're working on mathlib
```

Comment:
`cache-olean` is not limited to mathlib, you can use it on any Lean project. It won't fetch things from GitHub in random project but caches and fetches locally.